index.js

```diff
@@ -1,18 +1,79 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
-}
+const readline = require("readline"); {
+  const rl = readline.createInterface(process.stdin, process.stdout);
+
+  function ask(questionText) {
+    return new Promise((resolve, reject) => {
+      rl.question(questionText, resolve);
+    });
+  }
+
+  function randomNum(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  }
+  start();
+  async function start() {
+    console.log(
+      "Let's play a game where you (Joe) make up a number and I (computer) try to guess it."
+    );
+
+    let secretNum = await ask(
+      "What is your secret number?\nI won't peek, I promise...\n"
+    );
+    let min = 0; //min
+    let max = 100; //max
+    let contestantOne = randomNum(min, max); //computers guess
+
+    let answer = await ask(`Is the answer ${contestantOne}, "yes" or "no"?`);
```

Comment:
This await ask could use a catch similar like you did for the await ask on line 33. On line 42 you created an else statement that just states "input not recognized" and asks again. Doing this for like 26 would help your functionality in the game.